packages/block-library/src/image/image-editing/use-transform-image.js

```diff
@@ -13,10 +13,13 @@ function useTransformState( { url, naturalWidth, naturalHeight } ) {
 	const [ aspect, setAspect ] = useState();
 	const [ defaultAspect, setDefaultAspect ] = useState();
 
-	const imageCrossOrigin = useSelect( ( select ) => {
+	const mediaCrossOrigin = useSelect( ( select ) => {
 		const value = select( 'core/block-editor' ).getSettings()
-			.imageCrossOrigin;
-		return typeof value === 'string' ? value : undefined;
+			.mediaCrossOrigin;
+		return typeof value === 'string' &&
+			value.includes( '', 'anonymous', 'use-credentials' )
```

Comment:
This looks a bit backwards 🙂 
```suggestion
			[ '', 'anonymous', 'use-credentials' ].includes( value )
```